.github/TEST_FAILURE_REPORT_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "{{ env.TITLE }} ({{ date | date("YYYY-MM-DD") }})"
+labels: [type::bug, type::testing]
```

Comment:
Could we add `source::auto` so we can easily filter issues that come from automation?

Reply:
Oh nice touch!

Reply:
```suggestion
labels: [type::bug, type::testing, source::auto]
```